hikari/api/rest.py

```diff
@@ -1910,8 +1913,10 @@ async def delete_webhook(
     @abc.abstractmethod
     async def execute_webhook(
         self,
-        webhook: snowflakes.SnowflakeishOr[webhooks.Webhook],
+        # TODO: update docs
+        webhook: typing.Union[snowflakes.Snowflakeish, webhooks.Webhook, guilds.PartialApplication],
```

Comment:
Executable webhook instead?

Reply:
Nah, that can stay as `webhook`. All webhooks are executable :)

Reply:
Executable Webhook is mixin for any entity that can be executed like a webhook (webhook and command interactions)

Reply:
A command interaction is basically a webhook. Tbh we can treat them as the same object